components/feature/pwa/src/main/java/mozilla/components/feature/pwa/feature/ManifestUpdateFeature.kt

```diff
@@ -63,6 +66,24 @@ class ManifestUpdateFeature(
     override fun start() {
         scope = MainScope()
         sessionManager.findSessionById(sessionId)?.register(this)
+        updadeUsageJob?.cancel()
+
+        updadeUsageJob = scope?.launch {
+            storage.updateManifestUsedAt(initialManifest.startUrl)
+        }
+    }
+
+    /**
+     * Updates manifest parameters like last time of usage.
+     */
+    @OnLifecycleEvent(Lifecycle.Event.ON_RESUME)
+    fun resume() {
+        scope = MainScope()
+        updadeUsageJob?.cancel()
+
+        updadeUsageJob = scope?.launch {
+            storage.updateManifestUsedAt(initialManifest.startUrl)
```

Comment:
Since its a one-month timescale, updating only when the PWA is started should be fine. We don't need to update on every pause and resume.